pkg/topology/arangodb/arangodb-peer-change.go

```diff
@@ -0,0 +1,20 @@
+package arangodb
+
+func (a *arangoDB) peerChangeHandler(obj *message.PeerStateChange) {
+	glog.Infof("><SB> peer change handler - update db with peer change message data")
+
+	// creates a new BGP peer document in Arango's BGPPeer vertex collection
+	bgp_peer_document := &arangoDB.BGPPeer{
```

Comment:
Please use camel case for variables names example: `bgpPeerDocument`

Reply:
done